CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ Breaking changes:
 
 New features:
 
+Bugfixes:
+
+Internal:
+
+## v0.14.2
+
+Breaking changes:
```

Comment:
Shall we not include the “breaking changes” header if there aren’t any?

Reply:
Yeah, let's drop it.